spec/takeaway_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'takeaway'
+
+describe Takeaway do
+  subject(:takeaway) { described_class.new(menu_klass = menu) }
+  let(:menu) {double('menu', :dishes => { "salad" => 12 }) }
+  let(:dish) {double :dish }
+  let(:dish2) {double :dish2}
+  let(:order) {double :order}
+
+  it { is_expected.to respond_to(:select).with(2).arguments }
+
+  context 'When selecting food items frome the menu' do
+
+    it 'selects items from the menu which is stored as an order' do
+      expect(takeaway.select("salad")).to eq "You have added 1 salad(s) to your basket"
```

Comment:
Metrics/LineLength: Line is too long. [87/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)